spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -52,7 +52,16 @@ var (
 
 func initTest(t *testing.T, id string) (instName, dbName string, cleanup func()) {
 	projectID := getSampleProjectId(t)
-	instName, cleanup = createTestInstance(t, projectID)
+	instName, cleanup = createTestInstance(t, projectID, false)
```

Comment:
It's better to pass the instance config name as a param instead of a boolean value.